docs/UserGuide.md

```diff
@@ -328,7 +333,7 @@ Example: `ls -d 2022-11-11`
 Shows a list of all tasks with matching names.
 
 Format: `ls -n KEYWORD*`
-* `KEYWORD`: One keyword or multiple, which should be separated by spaces.
+* `KEYWORD`: One keyword or multiple, which should be alphanumeric and separated by spaces.
```

Comment:
To be consistent with the command format given above, maybe we should use "letters and numbers" instead of alphanumeric (sounds a bit less cheem also)